pandas/tests/frame/test_query_eval.py

```diff
@@ -704,7 +704,7 @@ def setup_class(cls):
         super().setup_class()
         cls.engine = "numexpr"
         cls.parser = "python"
-        cls.frame = TestData().frame
+        cls.frame = tm.makeDataFrame()
```

Comment:
Should the rhs here not be a fixture? I vaguely recall this being used elsewhere as one if you can check

Reply:
That was also my first take but then I realised this would break the function signature compared to the base class. Which solution do you prefer?

Reply:
these *should* be just float_frame as a fixture; you need to inject it into *all* the tests that need it rather than doing like this.

Reply:
might want to revert this and change this file in a dedicated PR.

Reply:
Makes sense - I'll do just that